plugins/kernel/pyparts/memory.py

```diff
@@ -9,10 +10,39 @@
     VMSTAT,
     BUDDY_INFO,
     SLABINFO,
+    KernelChecksBase
 )
 
 YAML_PRIORITY = 1
 
+class KernelOOMChecks(KernelChecksBase, checks.EventChecksBase):
+
+    def __init__(self):
+        super().__init__(yaml_defs_label='memory-check')
+
+    def process_results(self, results):
+        """ See defs/events.yaml for definitions. """
+        info = {}
+        for defs in self.event_definitions.values():
+            for label in defs:
+                _results = results.find_by_tag(label)
+                if label == "oom":
+                    if _results:
+                        first_process_oomd = _results[0].get(3)
+                        time_oomd = _results[0].get(1) + " " +  _results[0].get(2)
+                        msg = ("oom-killer had been invoked, first for process ")
```

Comment:
please use formatted string like:

process_name = _results[0].get(3)
"oom-killer invoked for process '{}' at {} {}".format(process_name, _results[0].get(1), _results[0].get(2))